source/backend/tests/unit/api/Models/CompensationFinancialReportModelTest.cs

```diff
@@ -108,7 +108,7 @@ public void CompensationFinancialReportModel_FileNumberAndName_Null()
             var model = new CompensationFinancialReportModel(testFinancial, new CompensationFinancialReportTotalsModel(new List<PimsCompReqFinancial>()), new System.Security.Claims.ClaimsPrincipal());
 
             // Assert
-            model.AcquisitionNumberAndName.Should().Be(string.Empty);
+            model.FileNumberAndName.Should().Be(string.Empty);
```

Comment:
NIT: For this to be empty, the Acquisition and the Lease should be null, please include the Lease = null in the arrange just to be explicit.